### ISSUE TITLE
mage2vs: MAGENTO_STORE_ID is never defined

### ISSUE THREAD

**Issue by benjick** [DivanteLtd/vue-storefront-api#179]
When running `yarn mage2vs import` my script ends up failing on this URL: `https://foo.com/rest/V1/products-render-info?searchCriteria=&[....]&storeId=1&currencyCode=USD`
The `storeId` and `currencyCode` comes from `node_modules/mage2vuestorefront/src/config.js` which can be overridden by `process.env` but that's never done in `mage2vs.js` and it falls back to these erroneous values.

My config:
```
"availableStores": [
  "se"
],
"storeViews": {
  "multistore": true,
  "mapStoreUrlsFor": [
    "se"
  ],
  "se": {
    "storeId": 9,
    "name": "Swedish Store",
    "url": "/",
    "elasticsearch": {
      "host": "localhost:8080/api/catalog",
      "index": "vue_storefront_catalog_se"
    },
    // etc
```

**Comment by boehsermoe** [comment]
I have the same problem. My temporary fix was to add this `magentoConfig.MAGENTO_STORE_ID = storeView.storeId;` in line 132 of mage2vs.js.

```
    if (cmd.storeCode) {
      const storeView = config.storeViews[cmd.storeCode]
      if (!storeView) {
        console.error('Wrong storeCode provided - no such store in the config.storeViews[storeCode]', cmd.storeCode)
        process.exit(-1)
      } else {
        magentoConfig.INDEX_NAME = storeView.elasticsearch.index;
        magentoConfig.MAGENTO_STORE_ID = storeView.storeId;
      }
    }
```

**Comment by pkarw** [comment]
@boehsermoe can You please open a PR with that?

**Comment by lukeromanowicz** [comment]
Merged to master in release 1.8.3
